### PR TITLE
Add strict mode script

### LINKED ISSUE #7: Add bash script with common strict setup code
Currently the same stanza is duplicated between every bash script that enables a "strict mode" for the bash script to run under. It would be better for this to be globally available so that it can be included in every bash script instead of duplicated.

### PR DESCRIPTION
**Pull request by dkubb**
This branch adds a script that enables strict mode within bash scripts.

[Fix #7]

### CODE REVIEW

**Comment by dkubb** [comment]
@mbj this is ready for review.

File: alpine-ruby/sbin/strict-mode.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# Reference:
+# http://www.davidpashley.com/articles/writing-robust-shell-scripts/
+# http://kvz.io/blog/2013/11/21/bash-best-practices/
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+
+set -o errexit    # Exit when an expression fails
+set -o pipefail   # Exit when a command in a pipeline fails
+set -o nounset    # Exit when an undefined variable is used
+set -o noglob     # Disable shell globbing
+set -o noclobber  # Disable automatic file overwriting
```
Comment (mbj) [inline alpine-ruby/sbin/strict-mode.sh:12]: While I like the verbose explanations I dislike 5 calls to `set`. What about using the following format:

```
set \
  - o errexit \ # documentation
  - o other_flag \ # more docs
```

?

Comment (dkubb) [inline alpine-ruby/sbin/strict-mode.sh:12]: I'd probably align things a bit better, but I could test something similar.

Comment (mbj) [inline alpine-ruby/sbin/strict-mode.sh:12]: This was to demonstrate the idea, I did not spend the time on alignment as I assumed you'd do it.

Comment (dkubb) [inline alpine-ruby/sbin/strict-mode.sh:12]: It looks like you can't use a `#` comment after `\`.

I think the comments make things easier to understand, so I'll keep the 5 `set` calls for now.

Comment (mbj) [inline alpine-ruby/sbin/strict-mode.sh:12]: Also `set` is a buildin and so has not a big overhead when called. +1

**Pull request merged**